blocks/covid-19-accordion/covid-19-accordion.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Creates a banner with the extracted leading text and the picture element
+ * @param pictureElement
+ * @param extractedInnerHTML
+ * @returns {HTMLDivElement}
+ */
+function createBanner(pictureElement, extractedInnerHTML) {
+  const banner = document.createElement('div');
+  banner.classList.add('banner');
+
+  const info = document.createElement('div');
+  info.classList.add('info');
+  info.innerHTML = extractedInnerHTML;
+  banner.appendChild(info);
+
+  const icon = document.createElement('div');
+  icon.classList.add('icon');
+  icon.innerHTML = pictureElement.outerHTML;
+  banner.appendChild(icon);
+
+  return banner;
+}
+
+/**
+ * Creates dividers with close buttons for the expanded content of accordion
+ * @returns {HTMLDivElement}
+ */
+function createDividerWithCloseButton() {
+  const newDiv = document.createElement('div');
+  newDiv.classList.add('covid-19-accordion', 'custom-divider');
+
+  const hr = document.createElement('hr');
+  newDiv.appendChild(hr);
+
+  const closeButton = document.createElement('button');
+  closeButton.classList.add('covid-19-accordion', 'close-button');
+  // empty as we fill in from after pseudo element
+  closeButton.innerHTML = '';
+  closeButton.addEventListener('click', () => {
+    const accordion = document.querySelector('.covid-19-accordion .container .details');
+    const button = document.querySelector('#expand-accordion');
+
+    accordion.classList.remove('expanded');
+    button.setAttribute('aria-expanded', 'false');
+  });
+  newDiv.appendChild(closeButton);
+
+  return newDiv;
+}
```

Comment:
Suggestion (not thoroughly tested):
I would write this like so:

```
/**
 * Creates dividers with close buttons for the expanded content of accordion
 * @returns {HTMLDivElement}
 */
function createDividerWithCloseButton(block) {
  function closeAccordion() {
    const accordion = block.querySelector('.container .details');
    const button = block.querySelector('#expand-accordion');

    accordion.classList.remove('expanded');
    button.setAttribute('aria-expanded', 'false');
  }
  
  // shameless plug :) 
  return (
    div({ class: 'covid-19-accordion custom-divider' },
      domEl('hr'),
      button({ class: 'covid-19-accordion close-button', onclick: closeAccordion }),
    )
  );
}
```

1. pass the `block` as a parameter to not look in the whole document and have all `querySelector` calls scoped to your block. In theory you can have the same block multiple times in a page. You don't want them interfering with eachother

2. the `dom-helpers` (for me personally) make the code look more intuitive, so next time when you come to read it you see what the resulting HTML will look like.